.circleci/config.yml

```diff
@@ -149,7 +149,7 @@ jobs:
       - *attach_workspace
```

Comment:
The job is called `integration_ng5_tests`, could you rename to `integration_ng6_tests`?